src/main/java/io/zeebe/monitor/rest/ServiceStatusViewController.java

```diff
@@ -0,0 +1,62 @@
+package io.zeebe.monitor.rest;
+
+import io.camunda.zeebe.client.api.response.BrokerInfo;
+import io.camunda.zeebe.client.api.response.PartitionInfo;
+import io.camunda.zeebe.client.api.response.Topology;
+import io.zeebe.monitor.rest.dto.BrokerDto;
+import io.zeebe.monitor.rest.dto.PartitionInfoDto;
+import io.zeebe.monitor.rest.dto.StatusDto;
+import io.zeebe.monitor.zeebe.ZeebeStatusService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.info.BuildProperties;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.Map;
+
+@Controller
+public class ServiceStatusViewController extends AbstractViewController {
+
+  @Autowired private ZeebeStatusService zeebeStatusService;
+  @Autowired BuildProperties buildProperties;
+
+  @GetMapping("/views/service-status")
+  public String index(final Map<String, Object> model, final Pageable pageable) {
```

Comment:
:x: We could build the service status as a push notification instead of introducing a new page that queries the status.

If we get the status via push notification then we could see the status change immediately instead of checking the page or keeping it open. The detailed information could be hidden in a modal. 

The push notification could be similar to the workflow instance notifications.

In the UI, this could look for example like this:

![image](https://user-images.githubusercontent.com/4305769/155980563-73f61697-dd4d-4129-8da0-7adc7599ec92.png)

* the status is rendered on the header (green/red)
* the status is updated via push notifications
* a click on the status opens the details:

![image](https://user-images.githubusercontent.com/4305769/155980607-9c1ac4e8-fddb-41cf-9e87-6ee937b84947.png)

Reply:
implemented, as suggested.

See screenshots below (show healthy modal, detected change in orange, unhealthy modal)

<img width="1800" alt="Screenshot 2022-10-05 at 23 38 43" src="https://user-images.githubusercontent.com/1189394/194169004-2f14a55b-34ee-4b16-8c92-dec0c3e6802c.png">
<img width="1800" alt="Screenshot 2022-10-05 at 23 38 34" src="https://user-images.githubusercontent.com/1189394/194169007-7622440e-eea2-4e60-aac5-9d6f6f8c5207.png">
<img width="1798" alt="Screenshot 2022-10-05 at 23 38 19" src="https://user-images.githubusercontent.com/1189394/194169011-1f7f2c7e-8261-4950-926e-05aa8c00fa37.png">